debugview/src/debug/kotlin/com/revenuecat/purchases/debugview/InternalDebugRevenueCatBottomSheet.kt

```diff
@@ -0,0 +1,35 @@
+package com.revenuecat.purchases.debugview
+
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import kotlinx.coroutines.launch
+
+@OptIn(ExperimentalMaterial3Api::class)
```

Comment:
The bottom sheet APIs I'm using are marked as experimental, so I'm forced to opt in. This means there might be breaking changes at any point :(.

I thought about using `Material` instead of `Material3` to do this, but the paradigm is quite different... In `Material`'s version, you need to pass the UI below the bottom sheet as the `content` of the composable. IMO, that's not ideal, so I understand the new architecture they are using in `Material3`. In the end, I decided to go with `Material3`'s version since it provides a much easier to use API and more modern UI as well.

Reply:
This is a debug thing so I think it's fine to do this.